client/templates/home.html

```diff
@@ -1,7 +1,13 @@
 <template name="home">
 
     {{#if currentUser}}
-      {{> meetingSeriesList}}
+        <ul class="nav nav-tabs">
+            <li id="tab_meetings" data-template="meetingSeriesList" role="presentation" class="{{isTabActive 'tab_meetings'}}"><a href="#">Meetings</a></li>
+            <li id="tab_actionItems" data-template="actionItemList" data-action="actionItems" role="presentation" class="{{isTabActive 'tab_actionItems'}}"><a href="#">Action Items</a></li>
+        </ul>
+        <div class="active">
+            {{> Template.dynamic template=tab}}
+        </div>
     {{else}}
 
```

Comment:
Please remove the `href="#"`from your links. Because these break our tabs on InternetExplorer11. Michael removed all `href="#"` on 2017-09-14 with commit 141b7a21dc4543bf78907919e9168de2e0685bb1